### DISCUSSION TITLE
Read only repositories

### DISCUSSION THREAD

**Discussion by dirk2011** [garethgeorge/backrest#208] [Ideas]
I have made a bash linux script to make some backups with restic. Besides making backups its does do check, prune, etc operations.
I don't need backrest to make backups. But I would like to browse my backups with backrest. Have given backrest access to the restic repositories. But I get a error that the files are readonly. 
Would be great to use backrest for browsing.

**Comment by garethgeorge** [comment]
Hey, would you mind attaching any logs you have available? What action are you trying when you get a permission error?
